build-package.sh

```diff
@@ -672,6 +672,7 @@ for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
 		cd "$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX_CLASSICAL"
 		termux_step_post_massage
 		cd "$TERMUX_PKG_MASSAGEDIR"
+		cp -rpu "./$TERMUX_PREFIX_CLASSICAL" "$TERMUX_BASE_DIR"
```

Comment:
What are we actually copying here?

Reply:
Here the prefix (`./data/data/com.termux/files/usr/`) is copied, which has passed the “massage” stage, into the base Termux directory (`/data/data/com.termux/files/`). This was done so that changes by "massage" could affect the compilation of packages.

Reply:
Then what was the massage step doing previously if changes from it weren't getting put into the final package?

Reply:
Should it change files in system prefix? What would happen if the build it done in actual Android device?

Reply:
> So, if you decide to update the ncurses package and another package that depends on ncurses, then you will receive an error that those header files are missing.

Why? Not when installing through apt (and probably pacman) at least, why would the files be missing and why would the package manager complain about it?

Reply:
I created this PR after I received an error in the above mentioned story when updating packages to the Docker environment - https://github.com/termux-pacman/glibc-packages/actions/runs/9350633786/job/25734522819